src/applications/disability-benefits/all-claims/utils.jsx

```diff
@@ -820,3 +820,20 @@ export const DISABILITY_SHARED_CONFIG = {
     depends: hasNewDisabilities,
   },
 };
+
+export const isBDD = servicePeriods => {
```

Comment:
I debated which way to do it. The function I based this function off of that was grabbing the "most recent service period" passed in `formData.serviceInformation` so I just figured I'd pass in the specific data I needed, but it probably makes more sense to just pass in all the form data and then grab `servicePeriods`. That's what the majority of the functions in this file do. That way I can also verify that `serviceInformation` isn't `undefined` in the function. In my testing, it didn't seem like it could ever be `undefined`, but I'm sure there is some weird case where it can be, so it's probably best to check.